packages/engine/apache-arrow-js-bundle/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "apache-arrow-js-bundle",
+  "version": "0.0.0",
+  "private": "true",
+  "type": "module",
```

Comment:
In theory, we don’t need to use `.mjs` extension for package files if we have `type = module`. But it’s not wrong to have it. It’d be good to have them `src/index` and `webpack.config` to be aligned though.

Reply:
Good spot, some previous iteration caused me to need to swap to mjs I think. 
It isn't necessary now though, have swapped back